tests/tests/lexer_modes.rs

```diff
@@ -1,8 +1,7 @@
-use logos::Logos;
+use logos::Logos as _;
```

Comment:
This doesn't look right. The `logos::Logos` is a trait, while the `logos_derive::Logos` is a derive macro, they _can_ and _should_ coexist in the same namespace.

Reply:
Heh, you are right, weird as this is...